README.md

```diff
@@ -11,24 +8,23 @@ A Neural Net providing a subgrid parameterization of atmospheric convection in a
 The work is described in a [GRL paper](https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2020GL091363) with [accompanying code available](https://github.com/yaniyuval/Neural_nework_parameterization/tree/v.1.0.3). The repository contains the neural net and its implementation into a simple system for atmospheric modelling, [SAM](http://rossby.msrc.sunysb.edu/~marat/SAM.html).
 
 The aims of this repository are to:
-1. develop a standalone fortran module based on this neural net that can be used elsewhere,
+1. provide a standalone fortran module of the convection parameterisation based on this neural net that can be used elsewhere,
 2. deploy the module in another atmospheric model, and
-3. evaluate its performance.
+3. evaluate its performance (and provide instructions/code to do this).
 
-We may also perform an investigation into interfacing the pytorch implementation of the Neural Net using the [pytorch-fortran bridging code](https://github.com/Cambridge-ICCS/fortran-pytorch-lib) developed at the [Institute of Computing for Climate Science](https://cambridge-iccs.github.io/).
+We have also started to perform an investigation into interfacing the pytorch implementation of the Neural Net using the [pytorch-fortran bridging code](https://github.com/Cambridge-ICCS/fortran-pytorch-lib) developed at the [Institute of Computing for Climate Science](https://cambridge-iccs.github.io/).
 
-The model will first be deployed into the [Single Column Atmospheric Model (SCAM)](https://www.cesm.ucar.edu/models/simple/scam) - a single column version of the CESM.
-We plan to evaluate performance using SCAM in the gateIII configuration for tropical convection in a similar manner described by the [SCAM6 pulication in JAMES](https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2018MS001578).
-This will compare model performance to data from an intense observation period (IOP) described in an [AMS publication](https://journals.ametsoc.org/view/journals/atsc/36/1/1520-0469_1979_036_0053_saposs_2_0_co_2.xml).
+The model was first deployed into the [Single Column Atmospheric Model (SCAM)](https://www.cesm.ucar.edu/models/simple/scam) - a single column version of the CESM.
+We evaluate performance using Single Column CAM (SCAM) in the gateIII configuration for tropical convection in a similar manner described by the [SCAM6 pulication in JAMES](https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2018MS001578).
```

Comment:
More a note-to-self, we are going to move to focus on comparing to the TOGA II dataset which is a similar idea to GATE III run in SCAM. Documented in the same paper we already cite here.

Reply:
Do we want to mention it here?

Reply:
Yes, I think maybe mention both GATE III and TOGA II?

Reply:
Ah, just seen that you said it's the same paper

Reply:
added that to the readme